libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -1935,6 +1946,7 @@
 					};
 					CE4CE26A1C0E449C009F6029 = {
 						CreatedOnToolsVersion = 7.1;
+						LastSwiftMigration = 1100;
```

Comment:
@wmathurin Do you think this will cause problems with `Xcode 10.x`?

Reply:
Maybe.
Swift 5 works starting in 10.2. So we could use that (or 10.3).